Policies/github-acceptable-use-policies.md

```diff
@@ -17,7 +17,7 @@ Capitalized terms used but not defined in these Acceptable Use Policies have the
 You are responsible for using the Service in compliance with all applicable laws, regulations, and all of our Acceptable Use Policies. These policies may be updated from time to time and are provided below, as well as in our [Terms of Service](/articles/github-terms-of-service) and [Corporate Terms of Service](/articles/github-corporate-terms-of-service).
 
 ### 2. Content Restrictions
-Under no circumstances will Users upload, post, host, execute, or transmit any Content to any repositories that:
+Under no circumstances will Users upload, post, host, execute, or transmit any Content that:
```

Comment:
Removal of the scope changes the meaning to "it is prohibited to use GitHub by the people anyhow anywhere (not necessarily on GH) involved in these activities". Is it by design?

Reply:
How does this change deal with security research ex post facto? Since it was "allowed" when it was posted, is it allowed to stay? If allowed to stay, would making a minor modification invalidate the entire project?

Reply:
@KOLANICH I imagine the intended concern here is that GitHub hosts more than simple repositories, and they may be concerned about people hosting the content in pages, wikis, actions, packages, etc.

Reply:
It doesn't matter what we can imagine. It matters what is written.

Reply:
>I imagine the intended concern here is that GitHub hosts more than simple repositories, and they may be concerned about people hosting the content in pages, wikis, actions, packages, etc.

💯 - this is the intent behind the change. See https://docs.github.com/en/github/site-policy/github-terms-of-service#a-definitions for the definition of "content" on GitHub.com.